appinventor/appengine/src/com/google/appinventor/client/boxes/SourceStructureBox.java

```diff
@@ -33,7 +35,7 @@ public static SourceStructureBox getSourceStructureBox() {
   /**
    * Creates new source structure explorer box.
    */
-  private SourceStructureBox() {
+  public SourceStructureBox() {
```

Comment:
Note that by making this public and using it in the different YaProjectEditor configurations, it breaks the singleton contract of `getSourceStructureBox()`, which doesn't seem to be addressed.